src/overlays/actors/ovl_En_Fz/z_en_fz.h

```diff
@@ -6,9 +6,52 @@
 
 struct EnFz;
 
+typedef void (*EnFzActionFunc)(struct EnFz*, struct GlobalContext*);
+typedef void (*EnFzSpawnIceSmokeFunc)(struct EnFz*);
+
+typedef struct {
+    /* 0x0000 */ u8 freezingState; // 0,1,2: State of freezard (1 not freezing, 2 freezing)
+    /* 0x0001 */ u8 numFramesActive; // increments primAlphaState after reaching 7 (freezing), used in Gfx_TwoTexScroll
```

Comment:
This is a fairly common particle struct, and these are usually called `type` and `timer`, respectively.